apps/test/unit/lib/tools/jsdebugger/DebugConsoleTest.js

```diff
@@ -236,3 +237,137 @@ describe('The DebugConsole component', () => {
     });
   });
 });
+
+describe('The DebugConsole component with the react-inspector flag on', () => {
+  let root;
+
+  beforeEach(() => {
+    stubRedux();
+    registerReducers(reducers);
+    sinon.stub(experiments, 'isEnabled').returns(true);
+    expect(experiments.isEnabled('react-inspector')).to.equal(true);
+
+    root = mount(
+      <Provider store={getStore()}>
+        <DebugConsole showReactInspector={true} />
+      </Provider>
+    );
+  });
+
+  afterEach(() => {
+    experiments.isEnabled.restore();
+    restoreRedux();
+  });
+
+  const debugOutput = () => root.find('#debug-output');
+
+  describe('when input is coming from code workspace returns', () => {
+    it('an array input outputs an array without an arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: ['test'],
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('▶["test"]');
+    });
+
+    it('a string input outputs a string without an arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: 'hello world',
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('"hello world"');
+    });
+
+    it('an integer or mathematical operation input outputs an integer without an arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: 1 + 1,
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('2');
+    });
+
+    it('an object input outputs an object without an arrow', () => {
```

Comment:
NIT: "with an arrow"

Reply:
I'll refactor it! Thanks for catching that!